src/csharp/Intel/Generator/IdentifierConverter.cs

```diff
@@ -63,16 +63,104 @@ abstract class IdentifierConverter {
 	sealed class CSharpIdentifierConverter : IdentifierConverter {
 		public static IdentifierConverter Create() => new CSharpIdentifierConverter();
 		CSharpIdentifierConverter() { }
-		public override string Type(string name) => name;
-		public override string Field(string name) => name;
-		public override string EnumField(string name) => name;
-		public override string PropertyDoc(string name) => name;
-		public override string MethodDoc(string name) => name;
-		public override string Method(string name) => name;
-		public override string Constant(string name) => name;
-		public override string Static(string name) => name;
-		public override string Namespace(string name) => name;
-		public override string Argument(string name) => name;
+		public override string Type(string name) => Escape(name);
+		public override string Field(string name) => Escape(name);
+		public override string EnumField(string name) => Escape(name);
+		public override string PropertyDoc(string name) => Escape(name);
+		public override string MethodDoc(string name) => Escape(name);
+		public override string Method(string name) => Escape(name);
+		public override string Constant(string name) => Escape(name);
+		public override string Static(string name) => Escape(name);
+		public override string Namespace(string name) => Escape(name);
+		public override string Argument(string name) => Escape(name);
+
+		public static string Escape(string name) {
```

Comment:
This causes problems with XML docs, see `MemorySize.g.cs` / `UInt8` docs: `<see cref="@byte"/>` instead of `<see cref="byte"/>`. The `*Doc()` methods above should probably not call this method.